src/poliastro/tests/test_czml.py

```diff
@@ -70,8 +70,8 @@ def test_czml_add_orbit():
     "version": "1.0",
     "name": "document_packet",
     "clock": {
-        "interval": "2013-03-18T12:00:00.000/2013-03-18T23:59:35.108",
-        "currentTime": "2013-03-18T12:00:00.000",
+        "interval": "2013-03-18T12:00:00Z/2013-03-18T23:59:35Z",
+        "currentTime": "2013-03-18T12:00:00Z",
```

Comment:
I think there isn't any reason not to convert the rest of the properties (like ``epoch``) to the ISO8601 format.

Reply:
I'll update the commit to convert the rest of the properties to ISO8601 format.